docs/book/RELEASE

```diff
@@ -1,11 +1,25 @@
-To push the docs you must be an admin until we have proper CI around this.
-  
+*NOTE: There is an issue to fully automate the publishing of Gitbook updates:
+https://github.com/kubernetes-sigs/cluster-api/issues/655 For now these 
+instructions are still partially manually while we gain experience with
+autogenerated commits to ensure we do not publish broken documentation...*
+
+# Rebuild the Gitbook
+
 ```
-cd docs/book/
-gitbook install
-gitbook build
-cd ../../
+hack/build-gitbook.sh
+```
+
+# Verify that they appear to be correct
```

Comment:
> Verify the GitBook contents
?

Reply:
I'll add some more context. The intent is for the person updating the pages to view the rendered html and verify that it looks reasonable by looking for the expected changes. I am not sure if there is a good way to prove the documentation is as intended with code.